roles/reset/tasks/main.yml

```diff
@@ -195,7 +195,7 @@
       networking
       {%- endif %}
     state: restarted
-  when: ansible_os_family not in ["CoreOS", "Container Linux by CoreOS"]
+  when: ansible_os_family not in ["CoreOS", "Container Linux by CoreOS"] and reset_restart_network
```

Comment:
Do you have an issue where network restart doesn't work? We ran into issues with calico that are fixed upstream, but this was disabled (and not re-enabled) for normal calico deploy.

Reply:
I have some nodes that respond badly to restarting the network (device error from the NIC on bringing it back up).  So it was helpful to override it so I wasn't in IPMI all the time rebooting hosts and did not find any repercussions in our environment from skipping this.  Left if the default so as not to change other peoples flows that have better behaving hardware.

Reply:
Okay, so this is an issue completely related to the hardware and not Ubuntu/Calico/K8s. Works for me.